templates/skeleton/app/routes/cart.tsx

```diff
@@ -15,7 +15,7 @@ export async function action({request, context}: ActionFunctionArgs) {
 
   const [formData, customerAccessToken] = await Promise.all([
     request.formData(),
-    session.get('customerAccessToken'),
+    context.customerAccount.getAccessToken(),
```

Comment:
Not strictly necessary of a utility to add for this PR to work. But I feel this can provided a nice wrapper for the future if we ever change where `customerAccessToken` would be store.

Reply:
@blittle Just for my understanding, why do we purposeful chose to have a different session key than current?